channels/rdpecam/client/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+# FreeRDP: A Remote Desktop Protocol Implementation
+# FreeRDP cmake build script
+#
+# Copyright 2024 Oleg Turovski <oleg2104@hotmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+option(WITH_MS_RDPECAM_CLIENT "Build with video camera redirect client" OFF)
```

Comment:
I'd suggest to rework that so that it fall in the general schema for channels with `CHANNEL_RDPECAM_CLIENT`

Reply:
hmm, right, missed that.

then `OPTION_CLIENT_DEFAULT` should be `OFF` unless build on linux that would eliminate this variable.